src/configuration-parser.spec.ts

```diff
@@ -0,0 +1,77 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+let typewizConfig = '';
+const mockFs = {
+    readFileSync: jest.fn((filePath: string, options?: any) => {
+        if (filePath === 'src/typewiz.json' || filePath === path.resolve('not-found-file.json')) {
+            return fs.readFileSync(filePath, options);
+        } else {
+            return typewizConfig;
+        }
+    }),
+};
+jest.doMock('fs', () => mockFs);
+
+import { ConfigurationParser } from './configuration-parser';
+
+describe('ConfigurationParser', () => {
+    it('should throw an error if given non-existing typewiz.json file', () => {
```

Comment:
I'm not sure that it should explicitly throw an error when given a nonexistent file.

When there's no configuration, we should probably use some sane defaults.

Reply:
Makes sense, I'll default to an empty config (since this is allowed as well with a config file specified and the defaults are handled by the code using the config (`instrument.ts`, `apply-types.ts` and `compiler-helper.ts`)).